packages/xo-web/src/xo-app/settings/groups/index.js

```diff
@@ -153,11 +153,29 @@ export default class Groups extends Component {
     }
   }
 
+  _isLdapLoaded = createSelector(
+    () => this.props.plugins,
+    plugins =>
+      some(plugins, plugin => plugin.name === 'auth-ldap' && plugin.loaded)
+  )
+
   render() {
     const { groups, intl } = this.props
 
     return (
       <div>
+        {this._isLdapLoaded() && (
```

Comment:
Better disable the button instead. To not have a component in the UI which appears/disappears dynamically.

Reply:
It only depends on whether the plugin is loaded or not, so it shouldn't happen too often. On the other hand, if I make the button always visible, it would feel weird to people who don't (or can't) use the LDAP plugin, don't you think?

Reply:
You can add a tooltip to make it clear that it requires the plugin to be loaded. 

It's to avoid having component which appears suddenly when editing fields giving that the plugin can be loaded/unload at any moment.

Reply:
> You can add a tooltip to make it clear that it requires the plugin to be loaded.

But some (most?) users don't even have the plugin installed and some (most?) users don't have a LDAP server anyway. So this would add noise on the page for people who will probably never need it.

(However, it makes me realize that it would be a good idea to disable the button and add a tooltip when the plugin is installed/loaded but the groups synchronization isn't configured.)

> It's to avoid having component which appears suddenly when editing fields giving that the plugin can be loaded/unloaded at any moment.

I agree it's usually better to prevent the layout from moving too much by itself but in this case, I think it happens way to rarely to be an issue.

Reply:
In general, it's better to show disabled feature, because people can learn it exists without actively searching for it.

Also, not a lot of people are interested in groups, so only relevant people will be in this view.